src/styles/blocks/card.scss

```diff
@@ -0,0 +1,79 @@
+.card {
+  display: flex;
```

Comment:
`flex` is redundant for the card, each block element will be displayed on the new line by default